crates/process_info/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use std::path::PathBuf;
+
+mod linux;
+mod macos;
+
+#[derive(Debug, Clone)]
+pub struct ProcessInfo {
+    pub pid: u32,
+    pub name: String,
+    pub cwd: PathBuf,
+    pub argv: Vec<String>,
+}
+
+impl ProcessInfo {
+    // #[cfg(not(any(target_os = "macos", target_os = "linux", windows)))]
```

Comment:
Need to remove that too, and maybe add some docs around, at least a module-level (`//!`) one.